common/nicknames/wc_nicknames.txt

```diff
@@ -1,4 +1,111 @@
 ﻿###Historical
+nick_the_banshee = {}
+nick_twilight_of_the_gods = {}
+nick_the_sunblade = {}
+nick_the_vile = {}
+nick_the_burning_one = {}
+nick_the_emissary = {}
+nick_the_nightingale = {}
+nick_the_brute = {}
+nick_the_weaponmaster = {}
+nick_the_cunning = {}
+nick_the_unstoppable = {}
+nick_the_thunderous = {}
+nick_the_lightning_fist = {}
+nick_the_thunder_king = {}
+nick_the_mystery_seeker = {}
+nick_the_outcast = {}
+nick_the_redeemer = {}
+nick_the_decrepit = {}
+nick_the_blade = {}
+nick_the_corruptor = {}
+nick_the_calculator = {}
+nick_the_shatterer = {}
+nick_the_blightcaller = {}
+nick_the_summoner = {}
+nick_the_zealous = {}
+nick_the_ravager = {}
+nick_the_remorseless = {}
+nick_the_wildshaper = {}
 nick_the_prophet = {}
 nick_the_harbinger_of_the_void = {}
-nick_the_whacked = {}
\ No newline at end of file
+nick_the_ashbringer = {}
+nick_the_adamant = {}
+nick_the_australian = {}
+nick_the_lightbringer = {}
+nick_the_dire = {}
+nick_the_defiler = {}
+nick_the_sun_king = {}
+nick_lord_of_the_forest = {}
+nick_the_godslayer = {}
+nick_the_divine = {}
+nick_the_slitherer = {}
+nick_the_whacked = {}
+nick_the_darkener = {}
+
+### From Lifestyle
+nick_the_ice_hearted = {}
```

Comment:
Its okay as it is, but I think cold hearted would be more suitable than ice hearted. Its a more common English phrase. Of course, if there is a reason it should be ice hearted specifically, then leave it as is.
```suggestion
nick_the_cold_hearted = {}
```